<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>当前</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>世界</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>门派</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>队伍</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>指令</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTextBrowser" name="noticeMsgBrowser"/>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QTextBrowser" name="RoomMsgBrowser"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>362</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
